cumulusci.yml

```diff
@@ -92,6 +106,70 @@ tasks:
         options:
             sql_path: 'datasets/regression/test_data.sql'
             mapping: 'datasets/regression/mapping.yml'
+
+    ensure_record_type:
+        name: Ensure Campaign Record Types
+        description: This will ensure Record Types are enabled for the Campaign object in your org before installing Volunteers for Salesforce. If there are no Campaign record types yet, it will create one called Default.
+        class_path: cumulusci.tasks.salesforce.EnsureRecordTypes
+        options:
+            record_type_label: Default
+            record_type_developer_name: Default
+            sobject: Campaign
+
+    install_npsp_with_unpackaged_config:
+        description: Install NPSP with dependencies and unpackaged configuration
+        class_path: cumulusci.tasks.salesforce.UpdateDependencies
+        options:
+            dependencies:
+                - github: 'https://github.com/SalesforceFoundation/Cumulus'
+                - repo_name: Cumulus
+                  repo_owner: SalesforceFoundation
+                  subfolder: unpackaged/config/trial
+                  namespace_inject: 'npsp'
+                  unmanaged: False
+                - repo_name: Cumulus
+                  repo_owner: SalesforceFoundation
+                  subfolder: unpackaged/config/reports
+                  namespace_inject: 'npsp'
+                  unmanaged: False
+
+    insert_npsp_relationships:
```

Comment:
@davidmreed - just curious if there is a way to just call the npsp task so it's not in 2 places?

Reply:
From Cumulus's repo? No, we can't do that yet, although it's on the roadmap.